tests/python_tests/utils/ov_genai_pipelines.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import Enum
+from pathlib import Path
+
+from openvino_genai import SchedulerConfig, draft_model, ContinuousBatchingPipeline, LLMPipeline
+
+def dict_to_scheduler_config(scheduler_params: dict = None) -> SchedulerConfig:
+    scheduler_config = SchedulerConfig()
+    if scheduler_params is None:
+        scheduler_config.dynamic_split_fuse = True
+        # vLLM specific
+        scheduler_config.max_num_batched_tokens = 256
+        scheduler_config.max_num_seqs = 256
+
+        # Expedited number of blocks = text_blocks_n * G * n_prompts, where
+        # text_blocks_n - number of blocks required for storing prompt and generated text,
+        # currently it is 1 block for prompt (31 token with block_size 32) + 1 block for generated text (max length of generated text - 30 tokens);
+        # G - number of sequences in a sequence group, for beam search it is 2(group_size) * 3 (num_groups);
+        # n_prompts - number of prompts.
+        # For current parameters in tests expedited number of blocks is approximately 48.
+        scheduler_config.num_kv_blocks = 60
+    else:
+        for param, value in scheduler_params.items():
+            setattr(scheduler_config, param, value)
+
+    return scheduler_config
+
+
+class PipelineType(Enum):
+    STATEFUL = 1
+    STATELESS = 2
+    CONTINIOUS_BATCHING = 3
+    SPECULATIVE_DECODING = 4
+    PROMPT_LOOKUP_DECODING = 5
+
+
+def create_ov_pipeline(models_path: Path,
+                       pipeline_type: PipelineType = PipelineType.STATEFUL,
```

Comment:
default should be just `LLMPipeline` w/o extra arguments, which is CB on current master.